sdk/src/main/java/com/silanis/esl/sdk/builder/SignerBuilder.java

```diff
@@ -216,7 +218,7 @@ private Signer buildRegularSigner(){
         result.setId(id);
         result.setAttachmentRequirements(attachments);
         result.setKnowledgeBasedAuthentication(knowledgeBasedAuthentication);
-
+        result.setLocalLanguage(localLanguage);
```

Comment:
Do we need this local language field to use the local language? We set the local language as data in Role.
Why do we need duplicate settings?